publish.sh

```diff
@@ -28,34 +28,38 @@ for file in *; do
     cp $file.tar.gz x86_64_$file.tar.gz
 done
 
-# create a release
-release_id=$(curl -sL -X POST \
-    -H "Content-Type: application/json" \
-    -H "Accept: application/vnd.github.v3+json" \
-    -H "Authorization: token ${GITHUB_TOKEN}" \
-    -H "Cache-Control: no-cache" -d "{
-  \"tag_name\": \"v${VERSION}\",
-  \"target_commitish\": \"master\",
-  \"name\": \"v${VERSION}\",
-  \"body\": \"# \`qemu-*-static\` @ ${VERSION}\",
-  \"draft\": false,
-  \"prerelease\": false
-}" "https://api.github.com/repos/${REPO}/releases" | jq -r ".id")
-if [ "$release_id" = "null" ]; then
-    # get the existing release id
-    release_id=$(set -x; curl -sL \
-    -H "Content-Type: application/json" \
-    -H "Accept: application/vnd.github.v3+json" \
-    -H "Authorization: token ${GITHUB_TOKEN}" \
-    -H "Cache-Control: no-cache" \
-    "https://api.github.com/repos/${REPO}/releases" | jq -r --arg version "${VERSION}" '.[] | select(.name == "v"+$version).id')
-fi
+if [[ $TRAVIS_BRANCH == 'master' && $TRAVIS_PULL_REQUEST == 'false' ]]; then
```

Comment:
Thank you for your contribution, but I don't like this because you mixed logic.

Reply:
Please, see comment below.